all-products.html

```diff
@@ -11,59 +11,99 @@
+                            <h2>Product name</h2>
+                            <p>Price</p>
+                        </div>
+                    </div>
 
-				<!-- Third row of products -->
-				<div class="flex-container">
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
+                    <div class="product-box" id="first">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
 
-                </div>
-                
-                <div class="flex-container2">
-                    <div class="flex-child2"><h2>Product name</h2><p>Price</p></div>
-                    <div class="flex-child2"><h2>Product name</h2><p>Price</p></div>
-                    <div class="flex-child2"><h2>Product name</h2><p>Price</p></div>
-                    <div class="flex-child2"><h2>Product name</h2><p>Price</p></div>
-                </div>
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
 
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+
+                    <div class="product-box" id="first">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+
+                    <div class="product-box">
+                        <div class="image">Image will go here</div>
+                        <h2>Product name</h2>
+                        <p>Price</p>
+                    </div>
+                </div> 
+
+                <div class="nav-buttons">

```

Comment:
I also added next and previous buttons which allow the user to navigate between the product pages easier. Additionally, I added text that notifies the user of which page they currently are on.